From 7338f3f450d63f03e75abd72d3199bdbccedb27d Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Thu, 14 May 2009 21:16:23 -0700
Subject: Staging: cpc-usb: add TODO file

List what needs to be done to get this driver merged into
the main part of the kernel tree.

Cc: Sebastian Haas <haas@ems-wuensche.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/cpc-usb/TODO |    9 +++++++++
 1 file changed, 9 insertions(+)
 create mode 100644 drivers/staging/cpc-usb/TODO

--- /dev/null
+++ b/drivers/staging/cpc-usb/TODO
@@ -0,0 +1,9 @@
+Things to do for this driver to get merged into the main portion of the
+kernel:
+	- checkpatch cleanups
+	- sparse clean
+	- remove proc code
+	- tie into CAN socket interfaces if possible
+	- figure out sane userspace api
+
+Send patches to Greg Kroah-Hartman <greg@kroah.com>
